pkg/build/apis/build/v1/types.go

```diff
@@ -691,22 +691,6 @@ type SourceBuildStrategy struct {
 
 	// forcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty" protobuf:"varint,6,opt,name=forcePull"`
-
-	// runtimeImage is an optional image that is used to run an application
-	// without unneeded dependencies installed. The building of the application
-	// is still done in the builder image but, post build, you can copy the
-	// needed artifacts in the runtime image for use.
-	// Deprecated: This feature will be removed in a future release. Use ImageSource
-	// to copy binary artifacts created from one build into a separate runtime image.
-	RuntimeImage *kapi.ObjectReference `json:"runtimeImage,omitempty" protobuf:"bytes,7,opt,name=runtimeImage"`
-
-	// runtimeArtifacts specifies a list of source/destination pairs that will be
-	// copied from the builder to the runtime image. sourcePath can be a file or
-	// directory. destinationDir must be a directory. destinationDir can also be
-	// empty or equal to ".", in this case it just refers to the root of WORKDIR.
-	// Deprecated: This feature will be removed in a future release. Use ImageSource
-	// to copy binary artifacts created from one build into a separate runtime image.
-	RuntimeArtifacts []ImageSourcePath `json:"runtimeArtifacts,omitempty" protobuf:"bytes,8,rep,name=runtimeArtifacts"`
```

Comment:
@smarterclayton @bparees if we remove fields 7 and 8 from the protobuf spec, what happens if we add new fields in this struct in the future and reuse those numbers?

Reply:
discussed on IRC, will add a comment warning people not to reuse 7+8 until we get an answer on this from @smarterclayton or maybe @liggitt knows.

Reply:
you cannot reuse protobuf tags. add a tombstone comment like https://github.com/kubernetes/kubernetes/pull/44900/commits/d45a3423f08c95f285f90b7ee55db1f12ae4f9d2#diff-c5f41a1fd7d92b2773d2d5194a0fb6ffR2633

Reply:
thanks @liggitt, done.

Reply:
same goes for json fields, actually